app/styles/_overlay-forms.less

```diff
@@ -1,3 +1,7 @@
+.order-service-config-single-column {
+  width: 100%;
```

Comment:
should this get pushed up into catalog? especially since we are talking about switching the ordering overlay to single column